week1/homework/src/server.js

```diff
@@ -10,6 +10,36 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+
+    function calculate(response) {
```

Comment:
Try not to define functions inside of other functions. `calculate` here gets defined every time we call `http.createServer`.